helpers/filterSeats.js

```diff
@@ -21,8 +24,11 @@ module.exports = function(dust) {
     var venueProduct = _.head(params.venueProducts);
     var ticketRates = params.ticketRates;
     var roomRates = params.roomRates;
-    var bandColours = params.bandColours;
-    var sortByInput = params.sortBy;
+    var sortType = params.sortBy;
+    var seatLegend = params.seatLegend;
+
+    var bestPrice = {};
+    var currentPrice, topTicket;
```

Comment:
If we're declaring multiple variables within one 'var' declaration, can we be consistent and do all or none? e.g.
```
var bestPrice = {}.
    currentPrice,
    topTicket;
```